apps/challenges/urls.py

```diff
@@ -25,4 +25,7 @@
         name='get_challenges_based_on_teams'),
     url(r'(?P<challenge_pk>[0-9]+)/challenge_phase_split$', views.challenge_phase_split_list,
         name='challenge_phase_split_list'),
+    url(r'challenge/challenge_host_team/(?P<challenge_host_team_pk>[0-9]+)/zip/$',
```

Comment:
@taranjeet I think we we will need `challenge host team` as a parameter because it is related as a `Foreign Key` to `Challenge` model. What do you say?
Moreover, I will change the endpoint `zip` to more verbose name. :+1: